llvm/utils/lit/tests/shtest-env-positive.py

```diff
@@ -33,25 +33,24 @@
 # CHECK-NOT: # error:
 # CHECK: --
 
-# CHECK: PASS: shtest-env :: env-args-none.txt ({{[^)]*}})
+# CHECK: PASS: shtest-env :: env-calls-env.txt ({{[^)]*}})
+# CHECK: env env | {{.*}}
+# CHECK: # executed command: env env
+# CHECK-NOT: # error:
+# CHECK: --
+
+# CHECK: PASS: shtest-env :: env-no-subcommand.txt ({{[^)]*}})
 # CHECK: env | {{.*}}
 # CHECK: # executed command: env
 # CHECK: env FOO=2 BAR=1 | {{.*}}
 # CHECK: # executed command: env FOO=2 BAR=1
 # CHECK-NOT: # error:
 # CHECK: --
 
-# CHECK: PASS: shtest-env :: env-calls-env.txt ({{[^)]*}})
-# CHECK: env env | {{.*}}
-# CHECK: # executed command: env env
-# CHECK-NOT: # error:
-# CHECK: --
-
 # CHECK: PASS: shtest-env :: env-u.txt ({{[^)]*}})
 # CHECK: env -u FOO | {{.*}}
 # CHECK: # executed command: env -u FOO
 # CHECK-NOT: # error:
```

Comment:
Why did the `# CHECK: --` line get removed?

Reply:
I accidentally removed it in my last commit, but I added it back.